### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by DFurnes**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/8mLOWz).

### CODE REVIEW

**Comment by DFurnes** [comment]
Just checked over all of this and it seems good to me... lots of `namespace` reformatting, padding of `!`s in conditionals, and `array() → []` switcharoos.

File: app/Http/Controllers/AuthController.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ public function login(Request $request)
      */
     public function logout(Request $request)
     {
-        if (!$request->header('Session')) {
+        if (! $request->header('Session')) {
```
Comment (jonuy) [inline app/Http/Controllers/AuthController.php]: haha oh man, is this a thing? the space in front of the`!`? I kept changing this in Android code where I saw TouchLab do it.

File: app/Services/AWS.php [hunk 2]
```diff
@@ -17,17 +19,16 @@ class AWS
     public function storeImage($folder, $filename, $file)
     {
         if (is_string($file)) {
-            $path = 'uploads/' . $folder . '/' . $filename;
+            $path = 'uploads/'.$folder.'/'.$filename;
```
Comment (jonuy) [inline app/Services/AWS.php]: oh man and no spaces on the sides of a `.`. My code is ruined.

**Comment by jonuy** [comment]
:+1:

File: app/Http/Controllers/AuthController.php (hunk 1 above)
Comment (DFurnes) [inline app/Http/Controllers/AuthController.php]: Yeah, I'm not a huge fan either. That's how [Laravel](https://github.com/laravel/framework/blob/648add30883ccd91f5080bb231d8394bf39ba30f/src/Illuminate/Auth/Guard.php#L122) [framework](https://github.com/laravel/framework/blob/648add30883ccd91f5080bb231d8394bf39ba30f/src/Illuminate/Auth/Guard.php#L139) [code](https://github.com/laravel/framework/blob/648add30883ccd91f5080bb231d8394bf39ba30f/src/Illuminate/Auth/Guard.php#L159) seems to be written though. I'm happy with keeping or nixing this, no strong feelings.

File: app/Services/AWS.php (hunk 2 above)
Comment (DFurnes) [inline app/Services/AWS.php]: Hahaha, yeah. Another one I'm not a huge fan of. Again, up for keeping or nixing.

File: app/Http/Controllers/AuthController.php (hunk 1 above)
Comment (jonuy) [inline app/Http/Controllers/AuthController.php]: Nah I'm also for sticking with the style of the framework

Comment (DFurnes) [inline app/Http/Controllers/AuthController.php]: In Taylor we trust! :hammer:

**Pull request merged**